arbnode/batch_poster.go

```diff
@@ -1167,8 +1252,8 @@ func (b *BatchPoster) encodeAddBatch(
 		}
 		fullCalldata = append([]byte{}, method.ID...)
```

Comment:
where did the delay proofs go?

Reply:
It's still added to the args on line 1247.

Reply:
57*